buildSrc/subprojects/jvm/src/main/kotlin/gradlebuild/unittest-and-compile.gradle.kts

```diff
@@ -84,6 +84,9 @@ fun configureSourcesVariant() {
         main.groovy.srcDirs.forEach {
             outgoing.artifact(it)
         }
+        main.allSource.srcDirs.filter { it.name == "kotlin" }.forEach {
```

Comment:
Not as nice as for `java`/`groovy` - is there a better way?

Reply:
That's because we do not always apply the Kotlin plugin. Only to selected projects.

We could move this to `buildSrc/subprojects/uber-plugins/src/main/kotlin/gradlebuild/kotlin-library.gradle.kts` and add more artifacts to `transitiveSourcesElements` there.

You can then use the `gradlebuild.basics.accessors.kotlin` accessor there.